docs-en/chapter_data_structure/character_encoding.md

```diff
@@ -1,36 +1,36 @@
 # Character Encoding *
 
-In computers, all data is stored in binary form, and the character `char` is no exception. To represent characters, we need to establish a "character set" that defines a one-to-one correspondence between each character and binary numbers. With a character set, computers can convert binary numbers to characters by looking up a table.
+In the computer system, all data is stored in binary form, including characters (char). To represent characters, we need to develop a "character set" that defines a one-to-one mapping between each character and binary numbers. With the character set, computers can convert binary numbers to characters by looking up the table.
 
 ## ASCII Character Set
 
-The "ASCII code" is one of the earliest character sets, officially known as the American Standard Code for Information Interchange. It uses 7 binary digits (the lower 7 bits of a byte) to represent a character, allowing for a maximum of 128 different characters. As shown in the figure below, ASCII includes uppercase and lowercase English letters, numbers 0 ~ 9, some punctuation marks, and some control characters (such as newline and tab).
+The "ASCII code" is one of the earliest character sets and stands for American Standard Code for Information Interchange. It uses 7 binary digits (the lower 7 bits of a byte) to represent a character, allowing for a maximum of 128 characters. As shown in the figure below, ASCII includes uppercase and lowercase English letters, numbers 0 ~ 9, various punctuation marks, and certain control characters (such as newline and tab).
```

Comment:
I think the original is better:
"The "ASCII code" is one of the earliest character sets, officially known as the American Standard Code"
and
"It uses 7 binary digits (the lower 7 bits of a byte) to represent a character, allowing for a maximum of 128 characters."

optional:
"It uses 7 binary digits" can be changed to -> "It employs 7 binary digits"